test-suite/non-functional-tests/test-scenario-1/dockerfile

```diff
@@ -0,0 +1,56 @@
+FROM openjdk:17-jdk-alpine
+
+ARG JMETER_VERSION="5.5"
+ARG JMETER_HOME="/opt/apache-jmeter-${JMETER_VERSION}"
+ARG JMETER_BIN="${JMETER_HOME}/bin"
+ARG MIRROR_HOST=http://mirrors.ocf.berkeley.edu/apache/jmeter
+ARG JMETER_DOWNLOAD_URL="${MIRROR_HOST}/binaries/apache-jmeter-${JMETER_VERSION}.tgz"
+ARG JMETER_PLUGINS_DOWNLOAD_URL="http://repo1.maven.org/maven2/kg/apc"
+ARG JMETER_LIB_FOLDER="${JMETER_HOME}/lib/"
+ARG JMETER_PLUGINS_FOLDER="${JMETER_HOME}/lib/ext/"
+ARG INBOUND_CERT_PATH="/jmeter-config/certs"
+
+ARG FACADE_URL 
+ARG FACADE_PORT 
+ARG INBOUND_URL
+ARG INBOUND_PORT
```

Comment:
Minor, I'd have made these environment variables, thus preveting the need for a rebuild each time you wanted to test against a different host.

Reply:
completed these changes, makes sense the way you have explained